lib/io/hyped_spi/src/lib.rs

```diff
@@ -1,24 +1,30 @@
 #![no_std]
 
+pub mod cs;
+pub use cs::*;
```

Comment:
since there isn't much in `cs`, I would define it in-line here and then just `use hyped_spi::cs::HypedSpiCsPin` for wherever it is needed